.github/workflows/tests.yml

```diff
@@ -556,3 +581,8 @@ jobs:
 
     - name: Exercise RGW again
       run: ~/actionutils.sh headexec testrgw
+
+    - name: Print logs for failure
+      if: failure()
+      run: |
+        sudo snap logs microceph -n 1000
```

Comment:
Hm, I wish we could reduce the duplication of those print log stanzas here

Reply:
I am not sure if we can abstract this step out given, The actual step is a one-liner (and rest is step description in each run.).